pom.xml

```diff
@@ -472,6 +472,12 @@
             <version>0.6</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.entur.gbfs</groupId>
+            <artifactId>gbfs-java-model</artifactId>
+            <version>2.2.3-SNAPSHOT</version>
```

Comment:
I wanted to try out this code if it breaks with our input data but I cannot download this artifact. Can you use a released version, please?